tests/conftest.py

```diff
@@ -63,7 +63,7 @@ def pytest_addoption(parser):
         "--runslow",
         action="store_true",
         default=os.getenv("RUNSLOW", False),
-        help="run slow tests.",
+        help="run slow tests (legacy).",
```

Comment:
It's not just for legacy. There is a test that checks 80 or so pages for `not converted yet` messages in the American Government book.

Reply:
perhaps `(mostly legacy)`